### PR TITLE
fix(ua): change the User-Agent to use the new specs lib (version)

### PR DESCRIPTION
**Pull request by robertmogos**

### CODE REVIEW

File: src/components/__tests__/InstantSearch.js [hunk 1]
```diff
@@ -245,9 +246,10 @@ it('will call client.addAlgoliaAgent if present', () => {
     },
   });
 
-  expect(client.addAlgoliaAgent).toHaveBeenCalledTimes(1);
-  expect(client.addAlgoliaAgent.mock.calls[0][0]).toMatch(
-    /Vue InstantSearch \([a-z0-9.-]+\)/
+  expect(client.addAlgoliaAgent).toHaveBeenCalledTimes(2);
+  expect(client.addAlgoliaAgent).toHaveBeenCalledWith(`Vue (${Vue.version})`);
```
Comment (francoischalifour) [inline src/components/__tests__/InstantSearch.js]: I think you can chain the calls like so:

```js
expect(client.addAlgoliaAgent).toHaveBeenCalledWith([
  `Vue (${Vue.version})`,
  `Vue InstantSearch (${version})`
]);
```

Comment (robertmogos) [reply src/components/__tests__/InstantSearch.js]: Nope, it doesn't work

Comment (francoischalifour) [reply src/components/__tests__/InstantSearch.js]: Sorry, I meant:

```js
expect(client.addAlgoliaAgent).toHaveBeenCalledWith(
  `Vue (${Vue.version})`,
  `Vue InstantSearch (${version})`
);
```

Comment (robertmogos) [reply src/components/__tests__/InstantSearch.js]: That one neither :)

**Pull request merged**